docs/source/en/pipeline_tutorial.mdx

```diff
@@ -12,21 +12,21 @@ specific language governing permissions and limitations under the License.
 
 # Pipelines for inference
 
-The [`pipeline`] makes it simple to use any model from the [Model Hub](https://huggingface.co/models) for inference on a variety of tasks such as text generation, image segmentation and audio classification. Even if you don't have experience with a specific modality or understand the code powering the models, you can still use them with the [`pipeline`]! This tutorial will teach you to:
+The [`pipeline`] makes it simple to use any model from the [Hub](https://huggingface.co/models) for inference on tasks such as text generation, image segmentation, audio classification, and multimodal tasks like visual question answering. Even if you don't have experience with a specific modality or familiar with the underlying code behind the models, you can still use them for inference with the [`pipeline`]! This tutorial will teach you to:
```

Comment:
Here the list makes it feel those are the only tasks supported (as it grows longer it feels like a list of examples but a comprehensive list). What about "on any tasks related to language, computer vision, speech, or even multimodal tasks."